README.md

```diff
@@ -1,3 +1,50 @@
 # terraform-module-servicebus-namespace
 
-A Terraform module for creating an Azure Service Bus namespace
+A Terraform module for creating an Azure Service Bus namespace.
+Refer to the following link for a detailed explanation of the Azure Service Bus namespace.
+
+[Azure Service Bus Namespace](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-messaging-overview)
+
+## Usage
+
+The following example shows how to use the module to create an Azure Service Bus namespace.
+
+```terraform
+module "servicebus-namespace" {
+  source              = "git@github.com:hmcts/terraform-module-servicebus-namespace?ref=servicebus_tf"
```

Comment:
I expect these changes to be merged at about the same time, I would rather we include this change

Reply:
If we merge the changes, the infra builds who use this module would break. Should we consider giving the teams sometime to import their modules before we merge the changes with master?